pkg/ebpf/maps/generic_map_test.go

```diff
@@ -572,3 +572,21 @@ func TestIterateWithPointerKey(t *testing.T) {
 	require.NoError(t, it.Err())
 	require.Equal(t, expectedNumbers, actualNumbers)
 }
+
+func TestGenericHashMapCanUseBatchAPI(t *testing.T) {
+	hash, err := ebpf.NewMap(&ebpf.MapSpec{
+		Type:       ebpf.Hash,
+		KeySize:    4,
+		ValueSize:  4,
+		MaxEntries: 10,
+	})
+	if err != nil {
+		t.Fatal(err)
+	}
+	t.Cleanup(func() { hash.Close() })
+
+	gm, err := Map[int32, int32](hash)
+	require.NoError(t, err)
+
+	require.True(t, gm.CanUseBatchAPI())
```

Comment:
do you need to specific minimal kernel version for this to pass?

Reply:
It is more appropriate to test that batch API is supported.